### DISCUSSION TITLE
teoria relazionale - 5

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#72] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/bac6590d-6f73-4631-9336-91ea13777a6b)

**Comment by IonutCicio** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/ddc71d25-1ce9-4f6c-b9b1-d25fc3138cbc)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/0f67d742-38fd-45ca-ad26-a1fd97e17f8b)

Mi sono dimenticato di scrivere $(C)^+_G = ABCE...$ ma si capisce lo stesso il punto.

**Comment by daromacarlo** [comment]
![Immagine 2024-02-04 170410](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/69183c26-df7a-46c0-9a4a-d4ab2ca9a5cc)
![Immagine 2024-02-04 170434](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/bf7d3791-749b-45d4-a77d-5e7091265aa4)
![Immagine 2024-02-04 170446](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/d0e9af69-7152-4e8b-8e6a-71b850b4b187)
![Immagine 2024-02-04 170506](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/b9754168-037b-4a50-836d-bbc9a6a03096)
![Immagine 2024-02-04 170525](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/7a2046ac-7cd1-48af-87b7-bb18aec91d55)
![Immagine 2024-02-04 170536](https://github.com/sapienzastudentsnetwork/bdi2324/assets/135441131/ab6d3b64-4b4e-4f0f-aeb7-6273a02e7ffe)

**Comment by Frisbon** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/84731341/4e2aa8b3-76a4-4783-bdf8-7c5b78e235ba)
